easybuild/tools/module_generator.py

```diff
@@ -204,23 +204,23 @@ def load_module(self, mod_name):
             # not wrapping the 'module load' with an is-loaded guard ensures recursive unloading;
             # when "module unload" is called on the module in which the depedency "module load" is present,
             # it will get translated to "module unload"
-            load_statement = ["module load %(mod_name)s"]
+            load_statement = [self.LOAD_TEMPLATE, '']
         else:
             load_statement = [self.conditional_statement("is-loaded %(mod_name)s", self.LOAD_TEMPLATE, negative=True)]
-        return '\n'.join([''] + load_statement + ['']) % {'mod_name': mod_name}
```

Comment:
We're not. The string returned by `self.conditional_statement` already contains a newline, and for a non-conditional load the `''` will make sure it's followed by a newline